modules/rodoo-about.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/index.adoc
+
+:_content-type: CONCEPT
+[id="run-once-about_{context}"]
+= About the {run-once-operator}
+
+{product-title} relies on run-once pods to perform tasks such as deploying a pod or performing a build. Run-once pods are pods that have a `RestartPolicy` of `Never` or `OnFailure`.
+
+Cluster administrators can use the {run-once-operator} to force a limit on the time that those run-once pods can be active. Once the time limit expires, the cluster will try to actively terminate those pods. The main reason to have such a limit is to prevent tasks such as builds to run for an excessive amount of time.
```

Comment:
IBM SG. Change to:"After the time limit expires, the cluster tries to actively..."